hub/apps/desktop/modernize/modernize-wpf-tutorial-4.md

```diff
@@ -444,6 +444,9 @@ To add a notification to the app:
 
 This exception is caused by the fact that the Contoso Expenses app doesn't yet have package identity. Some WinRT APIs, including the notifications API, require package identity before they can be used in an app. UWP apps receive package identity by default because they can only be distributed via MSIX packages. Other types of Windows apps, including WPF apps, can also be deployed via MSIX packages to obtain package identity. The next part of this tutorial will explore how to do this.
 
+> [!NOTE]
+> Starting WinRT 19041, MSIX packaged desktop applications should use the [OnActivated](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.application.onactivated) for handling toast activations. For more details [ToastNotification Class](https://docs.microsoft.com/uwp/api/Windows.UI.Notifications.ToastNotification)
```

Comment:
What about SSP? OnActivated won't work for those right since they may not be using XAML at all. They would need to use PushTrigger implementation.

Reply:
Please remove this. First, OnActivated is only available to UWP apps, not MSIX desktop apps (as it's part of UWP's app model and not WPF's). Second, this doc doesn't even talk about activation at all. Third, for MSIX apps we currently still recommend them to use Nitro since it activates in-process rather than performs an EXE activation.

We can link them to the main toast documentation for desktop apps to have one single place if you want to add anything: https://docs.microsoft.com/windows/uwp/design/shell/tiles-and-notifications/send-local-toast-desktop?tabs=msix-sparse